Dockerfile

```diff
@@ -130,26 +130,22 @@ ENV NEXT_TELEMETRY_DISABLED 1
 
 RUN addgroup --system --gid 1001 nodejs
 RUN adduser --system --uid 1001 nextjs
+USER nextjs
 
-RUN chown nextjs:nodejs /fides/clients
-
-# We need to copy everything so we can rebuild with the new configs if needed
-COPY --from=built_frontend /fides/clients /fides/clients
-
+# TODO: Optimize this to copy the pre-built Privacy Center client files.
```

Comment:
This'll need to be farmed out into a follow-up issue; no need to optimize that build quite yet, but we can definitely make some improvements to leverage some of the excellent stuff turborepo and nextjs make available